app/views/shared/_state_file_intercom.html.erb

```diff
@@ -1,5 +1,5 @@
 <button type="button" class="open-intercom button--intercom-blue button--intercom ">
-  <%= image_tag("intercom-chat.svg", "aria-hidden": "true", class: "") %>
+  <%= image_tag("intercom-chat-#{params[:us_state] || "us"}.svg", "aria-hidden": "true", class: "") %>
```

Comment:
This should use the helper method `state_code_for_page_style` from ApplicationController instead of directly accessing params[:us_state], since most of our pages don't have that param in their route anymore

Reply:
I actually had to use `state_code_for_page_style || "us"` so that there are no errors on the login-options page:
![image](https://github.com/user-attachments/assets/7c1b8486-5246-40ad-bc6d-b677f3318358)

Reply:
sorry - should have been clearer as that was what i meant, but glad we were on the same wavelength!